jms-reactive-client/src/main/java/edu/psu/activemq/MessageProcessor.java

```diff
@@ -58,9 +63,9 @@
 @Data
 public abstract class MessageProcessor {
 
-  private static final String AMQ_JOB_ID_PROP_NAME = "scheduledJobId";
-  private static final String DELIVERY_COUNT_PROP_NAME = "swe-delivery-count";
-  private static final String UNIQUE_ID_MDC_KEY = "uniqueId";
+  public static final String AMQ_JOB_ID_PROP_NAME = "scheduledJobId";
```

Comment:
Why are these now public?  They don't seem like they'd be useful in the actual ``MessageProcessor``.